### ISSUE TITLE
Mask Array every entry is 1.0?

### ISSUE THREAD

**Issue by janbolle** [seujung/SNAIL-gluon#2]
Hi,

the mask array is always completely 1.
If I uncomment it is a triangle again.

>             #Generate masking part
>             mask = np.full(shape=(logits.shape[1], logits.shape[2]), fill_value=1).astype('float')
>             mask = np.triu(mask, 1)
>             mask = np.expand_dims(mask, 0)
>             mask = np.repeat(mask, logits.shape[0], 0)
>             # np.place(mask, mask == 1, 0.0)
>             # np.place(mask, mask == 0, 1.0)
>             mask = nd.array(mask)

Is there any issue about the direction of the triangle?
